R/public_api.r

```diff
@@ -99,8 +99,8 @@ events <- function(limit = 10, releases = TRUE, archivals = TRUE) {
   } else {
     "archivals"
   }
-
-  "/-/" %>%
+browser()
```

Comment:
I get a bug here, and with other public API functions too: `events()` is now already a data.frame (why?!) so the printing method, that's supposed to operate on a list, doesn't work.

Reply:
Bc you removed the `...` from `fromJSON()`. Apparently it is needed there. Then probably it was ignored in the HTTP query.